### PR TITLE
Redirection auth add expired session

### PR DESCRIPTION
**Pull request by hichri-louay**

### CODE REVIEW

File: src/app/wallet/components/migration/migration.component.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { environment } from '@environments/environment';
 import { ActivatedRoute, Router } from '@angular/router';
 import { Clipboard } from '@angular/cdk/clipboard';
 import { GazConsumed } from '@app/config/atn.config';
+import { environment as env } from './../../../../environments/environment';
```
Comment (ksibisamir) [inline src/app/wallet/components/migration/migration.component.ts]: import { environment } from '@environments/environment';

C'est plus clean

File: src/app/wallet/components/receive/receive.component.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ import { WalletFacadeService } from '@core/facades/wallet-facade.service';
 import { AccountFacadeService } from '@app/core/facades/account-facade/account-facade.service';
 import { ShowNumbersRule } from '@app/shared/pipes/showNumbersRule';
 import { TokenStorageService } from '@app/core/services/tokenStorage/token-storage-service.service';
+import { environment as env } from './../../../../environments/environment';
```
Comment (ksibisamir) [inline src/app/wallet/components/receive/receive.component.ts]: import { environment } from '@environments/environment';

C'est plus clean

File: src/app/wallet/components/send/send.component.ts [hunk 3]
```diff
@@ -41,6 +41,7 @@ import { KycFacadeService } from '@app/core/facades/kyc-facade/kyc-facade.servic
 import { BarcodeFormat } from '@zxing/library';
 import { ActivatedRoute, Router } from '@angular/router';
 import { ITransferTokensRequestBody } from '@app/core/services/wallet/wallet.service';
+import { environment as env } from './../../../../environments/environment';
```
Comment (ksibisamir) [inline src/app/wallet/components/send/send.component.ts:45]: import { environment } from '@environments/environment';

c'est plus clean

File: src/app/wallet/components/receive/receive.component.ts (hunk 2 above)
Comment (hichri-louay) [reply src/app/wallet/components/receive/receive.component.ts]: @ksibisamir done

File: src/app/wallet/components/send/send.component.ts (hunk 3 above)
Comment (hichri-louay) [reply src/app/wallet/components/send/send.component.ts:45]: @ksibisamir done

File: src/app/wallet/components/migration/migration.component.ts (hunk 1 above)
Comment (hichri-louay) [reply src/app/wallet/components/migration/migration.component.ts]: @ksibisamir done

**Pull request merged**